### DISCUSSION TITLE
[Feature request] Auto-switch tabs

### DISCUSSION THREAD

**Discussion by SvenSvenApfel** [kinghat/tabbed-card#99] [Feature Requests]
Hello! The tabbed cards are awesome, thank you for that!

I have a feature that would be neat: I use your card to show 4 departure-tables for my local subway. I want to use this on a display and it would be nice if the tabs would change automatically like every 15 seconds or so so that I can see all tabs without having to install an input-device. 

Thanks! :)

**Comment by ron-osu** [comment]
I think a more universal way to implement this feature would be to allow an entity to be assigned in the tabbed-card configuration, most likely a helper such as input_text or input_number that corresponds to the current tab selection.

Two-way functionality:
- Manually switching tabs in the UI updates the associated helper's value
- Updating the associated helper's value outside the UI automatically switches the active tab in the UI

For my application... I have a tab for each camera or group of cameras.  When a person/vehicle is detected on a specific camera, an automation would update the helper's value to automatically switch to the correct tab in the UI.

While I absolutely love tabbed-card (thank you @kinghat!) - I am unfortunately in the process of trying to manually replicate the functionality using buttons that change the value of an input_text helper, and conditional cards that show/hide the associated WebRTC camera card based on the value of the helper.  Then of course, I can also change the helper's value outside the UI (in an automation/script) to show a specific camera.  It's functional but getting all the UI elements to work as well as tabbed-card is a losing battle.
